packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -154,13 +154,21 @@ information, see the States Language spec.
 
 ### Task
 
-A `Task` represents some work that needs to be done. It takes a `resource`
-property that is either a Lambda `Function` or a Step Functions `Activity`
-(A Lambda Function runs your task's code on AWS Lambda, whereas an `Activity`
-is used to run your task's code on an arbitrary compute fleet you manage).
+A `Task` represents some work that needs to be done. There are specific
+subclasses of `Task` which represent integrations with other AWS services
+that Step Functions supports. For example:
+
```

Comment:
I am concerned about the extremely overloaded term "Task" without any context to Step Functions. It will be a bit confusing to browse the lambda module and see this thing. It's not clear from the context what it is.
Maybe we can call this `StepTask` or even, god forbid, `StepFunctionsTask`